paasta_tools/utils.py

```diff
@@ -191,14 +200,21 @@ def check_cpus(self):
         cpus = self.get_cpus()
         if cpus is not None:
             if not isinstance(cpus, float) and not isinstance(cpus, int):
```

Comment:
isinstance can be passed a tuple for its second argument so this could just be `isinstance(cpus, (float, int))`

Ditto for line 209